packages/python/accern_data/util.py

```diff
@@ -89,6 +89,57 @@ def get_overall_total_from_dummy(
     return response_obj
 
 
+def generate_csv_object(
+        path: str,
+        date: pd.Timestamp,
+        harvested_after: pd.Timestamp,
+        filters: Dict[str, str],
+        encoding: str) -> io.BytesIO:
+    df = pd.read_csv(path)
+    df["harvested_at"] = pd.to_datetime(df["harvested_at"])
+    df["published_at"] = pd.to_datetime(df["published_at"])
+
+    valid_df: pd.DataFrame = df[
+        (df["published_at"] == date) &
+        (df["harvested_at"] > harvested_after)
+    ]
+    if valid_df.empty:
+        filtered_df = valid_df
+    else:
+        result = pd.Series(
+            [True for _ in range(valid_df.shape[0])], index=valid_df.index)
```

Comment:
No need to do a list with all true. Just do true

Reply:
https://stackoverflow.com/questions/58415637/initialize-all-true-boolean-index-for-pandas